utilities/rali/rali_performance_tests_with_depth/CMakeLists.txt

```diff
@@ -0,0 +1,72 @@
+################################################################################
```

Comment:
@paveltc the changes look good. I was wondering if we can change it to rali/performance_tests... so on, having rali/rali_ on all folders would be redundant. May be we can rename the folders without rali_.Let me know what you think.